Sources/TuistLoader/Models+ManifestMappers/TargetAction+ManifestMapper.swift

```diff
@@ -23,6 +23,14 @@ extension TuistGraph.TargetScript {
         let runForInstallBuildsOnly = manifest.runForInstallBuildsOnly
         let shellPath = manifest.shellPath
 
+        let dependencyFile: AbsolutePath?
+
+        if let manifestDependencyFile = manifest.dependencyFile {
+            dependencyFile = try absolutePaths(for: [manifestDependencyFile], generatorPaths: generatorPaths).first
+        } else {
+            dependencyFile = nil
+        }
```

Comment:
```suggestion
        let dependencyFile = manifest.dependencyFile.map { try absolutePaths(for: [$0], generatorPaths: generatorPaths).first
```

Reply:
This will produce a `AbsolutePath??` since `.first` is also an optional.